config/field_test.yml

```diff
@@ -28,10 +28,38 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
+  feed_strategy_starting_20220726:
+    # NOTE: Required as we want only want to consider for conversion events that
+    #       occurred on or after the given start_date.
+    started_at: 2022-07-26
+    variants:
+      - 20220719-variant-a
```

Comment:
looking at the spec sheet, shouldn't this be `20220719-variant-b` ?

Reply:
Ah, great catch, thanks!